acct-user/_bgpd/_bgpd-0.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+KEYWORDS="~amd64 ~x86"
+
+DESCRIPTION="User for OpenBGPD"
+# BGP port 179
+ACCT_USER_ID=179
+ACCT_USER_GROUPS=( _bgpd )
+ACCT_USER_HOME=/var/empty
```

Comment:
Definitely need to set this? http://michael.orlitzky.com/articles/gentoo_glep81_user_package_guidelines.xhtml

Reply:
I just took the value from an OpenBGP box, I could try with the default /dev/null if you prefer.

The directories have the same permissions between the OSes, except for the root/wheel group:
```
guinch# grep bgp /etc/passwd                                                                                                                                                         
_bgpd:*:75:75:BGP Daemon:/var/empty:/sbin/nologin
guinch# ls -lh /var/empty/                                                                                                                                                           
guinch# ls -lhd /var/empty/
drwxr-xr-x  2 root  wheel   512B Oct 12  2019 /var/empty/

alarig@pikachu ~ % ls -lh /var/empty/
total 0
alarig@pikachu ~ % ls -lhd /var/empty/
drwxr-xr-x 2 root root 4.0K Mar 30 08:38 /var/empty/
alarig@pikachu ~ % 
```